src/client/containers/DeleteBoardModal/DeleteBoardModal.container.js

```diff
@@ -22,8 +20,11 @@ const DeleteBoardModal = ({
         throw new Error(response.status);
       }
       alert('Data deleted');
-    } catch (error) {
-      deleteErrorHandler(error);
+    } catch {
+      setError(() => {
```

Comment:
You need to get the error from the catch here. 
catch (error) {

then pass the message and the status code to the ApiError